src/Microsoft.AspNetCore.Mvc.Razor.Extensions/Microsoft.AspNetCore.Mvc.Razor.Extensions.csproj

```diff
@@ -11,6 +11,16 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\Microsoft.AspNetCore.Razor.Language\BoundAttributeDescriptorComparer.cs">
+      <Link>BoundAttributeDescriptorComparer.cs</Link>
+    </Compile>
+  </ItemGroup>
```

Comment:
`Someone besides you and me doesn't have the ability to just include this file. What should that person do?`
I don't think they can use it either way because this is internal.